### ISSUE TITLE
current update regime breaks compiling

### ISSUE THREAD

**Issue by jonka-blip** [parcel-bundler/parcel#7572]
Since version 2.1.0 whenever a new version of parcel is released, my CD breaks. 

at the moment error is like this:

```
Building...
🚨 Build failed.

@parcel/package-manager: Could not find module "@parcel/transformer-sass"
satisfying ^2.2.1.

  /app/frontend/package.json:36:9
    35 |         "@parcel/transformer-css-experimental": "2.2.0",

  > 36 |         "@parcel/transformer-sass": "2.2.0",
  >    |         ^^^^^^^^^^^^^^^^^^^^^^^^^^ Found this conflicting local requirement.
    37 |         "autoprefixer": "10.4.2",
    38 |         "cssnano": "5.0.15",
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
1
Error: Process completed with exit code 1.
```
The parcel binary has version 2.2.0

My investigation with `yarn why` shows, that this due to the Caret Ranges in the dependencies of parcels packages (`^2.2.0`). 

Although I pinned all my version to 2.2.0, this leads to the installment of @parcel/package-manager 2.2.1, which then emerges this conflict.

To fix this I have to always update Parcel to the latest version, which is not what I want. 

thx and keep up the good work

**Comment by William-McGonagle** [comment]
Yep- broken for me too.

**Comment by jonka-blip** [comment]
got fixed with 2.3.1 👍
